cpp/modmesh/view/wrap_view.cpp

```diff
@@ -440,6 +439,95 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRManager
 
 }; /* end class WrapRManager */
 
+class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRCameraController
+    : public WrapBase<WrapRCameraController, CameraController>
+{
+
+    friend root_base_type;
+
+    WrapRCameraController(pybind11::module & mod, char const * pyname, char const * pydoc)
+        : root_base_type(mod, pyname, pydoc)
+    {
+        namespace py = pybind11;
+
+        (*this)
+            .def(
+                "updateCameraPosition",
+                [](
+                    wrapped_type & self,
+                    float const & x,
```

Comment:
Please use (non-const) `float` for arguments of fundamental types instead of `const &`.  A `float` takes 4 bytes and a `float const &` or `float const *` takes 8 bytes (pointer).

The generated assembly might be the same between `float` and `float const &` but the semantics differ.  Arguments of a const reference type cannot be reused in the callee.  But the most significant issue is the conflict with what we expect the compiler to do.

Reply:
It is unfortunate that clang-format forces you to use a single line per argument for this wrapping lambda.  I hope there is an option to ask clang-format to format like what I want:

```cpp
[](wrapped_type & self, float x, float y, float z, float pitch, float yaw,
   bool left_mouse_button, bool right_mouse_button,
   bool alt_key, bool shift_key, float dt)
```

But this is future work.  We'll follow what clang-format does for us even if it is not perfect.  It's more important to have a consistent style.